src/common/utxobased/info/ecash.ts

```diff
@@ -0,0 +1,83 @@
+import { Psbt } from 'altcoin-js'
+import { asCodec, asString } from 'cleaners'
+import { EdgeCurrencyInfo } from 'edge-core-js/types'
+
+import { CoinInfo, EngineInfo, PluginInfo } from '../../plugin/types'
+import { maximumFeeRateCalculator } from '../../plugin/util/maximumFeeRateCalculator'
+import { utxoCustomFeeTemplate, utxoMemoOptions } from './commonInfo'
+
+const currencyInfo: EdgeCurrencyInfo = {
+  currencyCode: 'XEC',
+  customFeeTemplate: utxoCustomFeeTemplate,
+  displayName: 'eCash',
+  memoOptions: utxoMemoOptions,
+  pluginId: 'ecash',
+  walletType: 'wallet:ecash',
+
+  // Explorers:
+  blockExplorer: 'https://blockchair.com/ecash/block/%s',
+  addressExplorer: 'https://explorer.e.cash/address/ecash:%s',
+  transactionExplorer: 'https://explorer.e.cash/tx/%s',
+
+  denominations: [
+    { name: 'XEC', multiplier: '100', symbol: 'e' },
+    { name: 'sats', multiplier: '1', symbol: 's' }
+  ],
+  defaultSettings: {
+    customFeeSettings: ['satPerByte'],
+    blockbookServers: ['wss://blockbook.fabien.cash/websocket'],
+    enableCustomServers: false
+  },
+  metaTokens: [],
+  chainDisplayName: '',
+  assetDisplayName: ''
```

Comment:
These should be set; `chainDisplayName: 'eCash'` and `assetDisplayName: 'eCash'` , I presume.

The field `displayName` has been deprecated for these two distinct fields. The `displayName` is a legacy field.

[See how we separate deprecated fields](https://github.com/EdgeApp/edge-currency-plugins/blob/master/src/common/utxobased/info/bitcoin.ts#L33) in the `EdgeCurrencyInfo` definition. Please move all deprecated fields below a `// Deprecated:` comment if you may.